assets/js/components/dashboard-details/DashboardDetailsHeader.js

```diff
@@ -90,22 +90,35 @@ export default function DashboardDetailsHeader() {
 								</Fragment>
 							) }
 							{ ! currentEntityURL && (
-								<p
-									dangerouslySetInnerHTML={ sanitizeHTML(
+								<p>
+									{ createInterpolateElement(
 										sprintf(
-											/* translators: %s: current entity URL */
+											/* translators: %s: current entity URL. */
 											__(
-												'It looks like the URL %s is not part of this site, therefore there is no data available to display.',
+												`It looks like the URL %s is not part of this site or is not based on standard WordPress content types, therefore there is no data available to display. Visit our <link1>support forums</link1> or <link2>website</link2> for support or further information`,
```

Comment:
I just noticed this text after reviewing the IB for #4287; but this text should be adjusted as the link "website" is not very accessible for a screen reader user tabbing through links on the page.

It would be better if the text for the second link (`"website"`) was changed to `<VisuallyHidden>Site Kit </VisuallyHidden> website` so that there is more context for a user tabbing through links.

Sorry to drive-by code review, but I wanted to point this out as it was a comment I had for #4287's ACs as well.

Reply:
Good catch, @tofumatt. Also we need to use single quotes for simple strings: 
```suggestion
												'It looks like the URL %s is not part of this site or is not based on standard WordPress content types, therefore there is no data available to display. Visit our <link1>support forums</link1> or <link2>website</link2> for support or further information',
```

Reply:
@tofumatt @eugene-manuilov thanks for pointing it out! But I am not sure how to handle the translation on that with createInterpolateElement/sprintf :face_exhaling: I cannot get the nested element to work in it. Any pointer? Would this work? 

`<VisuallyHidden>Site Kit </VisuallyHidden>  <a>website</a>`

Reply:
@kuasha420, try this:

```js
<p>
	{ createInterpolateElement(
		sprintf(
			/* translators: %s: current entity URL. */
			__(
				`It looks like the URL %s is not part of this site or is not based on standard WordPress content types, therefore there is no data available to display. Visit our <link1>support forums</link1> or <link2><VisuallyHidden>Site Kit </VisuallyHidden>website</link2> for support or further information`,
				'google-site-kit'
			),
			`<strong>${ permaLink }</strong>`
		),
		{
			strong: <strong />,
			link1: (
				<Link
					href="https://wordpress.org/support/plugin/google-site-kit/"
					external
					inherit
				/>
			),
			link2: (
				<Link
					href="https://sitekit.withgoogle.com/"
					external
					inherit
				/>
			),
			VisuallyHidden: <VisuallyHidden />,
		}
	) }
</p>
```